mango-cli/src/main/webapp/stylesheets/main.css

```diff
@@ -151,3 +151,67 @@ input[type="text"] {
   pointer-events: none;
   cursor: default;
   color: grey; }
+
+
+#searchfield { display: block; width: 100%; text-align: center; margin-bottom: 35px; }
+
+#searchfield form {
+  display: inline-block;
+  background: #eeefed;
```

Comment:
Ok I've made the changes to main.scss instead and compiled with Koala on my branch, as you suggested. The pull request is merged so it isn't updating. I think it can go in next time I do a pull request.